audiobook/main.py

```diff
@@ -88,6 +84,8 @@ def create_json_book(self, input_book_path, password=None, extraction_engine=Non
             json_book, metadata = html_to_json(input_book_path)
         elif input_book_path.endswith((".docx", ".doc")):
             json_book, metadata = docs_to_json(input_book_path)
+        else:
+            raise NotImplementedError("Only PDF, TXT, EPUB, MOBI, HTTP, DOCX and DOC files are supported")
```

Comment:
Had to add an `else` condition to avoid errors saying that `json_book` and `metadata` are not always specified